test/e2e/index.test.ts

```diff
@@ -117,16 +69,49 @@ describe('e2e', () => {
       await renderFixture('Radio');
 
       await page.keyboard.press('Tab');
-      await expect(screen.getByTestId('one')).toHaveFocus();
+      await expect(page.getByTestId('one')).toBeFocused();
 
       await page.keyboard.press('ArrowRight');
-      await expect(screen.getByTestId('two')).toHaveFocus();
+      await expect(page.getByTestId('two')).toBeFocused();
 
       await page.keyboard.press('ArrowLeft');
-      await expect(screen.getByTestId('one')).toHaveFocus();
+      await expect(page.getByTestId('one')).toBeFocused();
 
       await page.keyboard.press('ArrowLeft');
-      await expect(screen.getByTestId('three')).toHaveFocus();
+      await expect(page.getByTestId('three')).toBeFocused();
+    });
+  });
+
+  describe('<Slider />', () => {
+    it('overlapping thumbs', async () => {
+      await renderFixture('RangeSlider');
+
+      // mouse down at the center of the lower thumb but the upper thumb
+      // is moved due to overlap
+      await page.mouse.move(25, 10);
+      await page.mouse.down();
+      await page.mouse.move(100, 10);
+      await page.mouse.up();
+
+      const output = page.getByRole('status');
+      const outputText = await output.evaluate((el) => el.textContent);
+      expect(outputText).toBe('25 – 100');
```

Comment:
Nitpick: WDYT about writing it this way? 🤔 
The idea: avoid dropping to JS level if possible.
```suggestion
      expect(page.getByRole('status', { name: '25 – 100' })).toBeVisible();
```

Reply:
It seemed `evaluate` was the only way to avoid timeouts but `await expect(page.getByRole('status')).toHaveText('25 – 100')` works too (`toBeVisible` also times out here I dunno why)

Reply:
Cool, that's even better/cleaner. 👍 
For some reason `toHaveText` was throwing errors to me locally. 🙈